maybe-uninit warning in tests.c

With gcc-10 under `-O3` I get:
```
src/tests.c: In function ‘test_ecmult_multi_batch_single.constprop’:
src/tests.c:2716:9: warning: ‘sc’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2716 |     *sc = data->sc[idx];
      |     ~~~~^~~~~~~~~~~~~~~
src/tests.c:2717:9: warning: ‘pt’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2717 |     *pt = data->pt[idx];
      |     ~~~~^~~~~~~~~~~~~~~
src/tests.c:2717:9: warning: ‘*((void *)&pt+16)’ may be used uninitialized in this function [-Wmaybe-uninitialized]
src/tests.c:2717:9: warning: ‘pt.x.n[4]’ may be used uninitialized in this function [-Wmaybe-uninitialized]
src/tests.c:2717:9: warning: ‘*((void *)&pt+40)’ may be used uninitialized in this function [-Wmaybe-uninitialized]
src/tests.c:2717:9: warning: ‘*((void *)&pt+48)’ may be used uninitialized in this function [-Wmaybe-uninitialized]
src/tests.c:2717:9: warning: ‘*((void *)&pt+64)’ may be used uninitialized in this function [-Wmaybe-uninitialized]
src/tests.c:2717:9: warning: ‘pt.y.n[4]’ may be used uninitialized in this function [-Wmaybe-uninitialized]
src/tests.c:2717:9: warning: ‘*((void *)&pt+88)’ may be used uninitialized in this function [-Wmaybe-uninitialized]
src/tests.c:2717:9: warning: ‘pt.infinity’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  CCLD     tests
```

--- a/src/tests.c
+++ b/src/tests.c
@@ -2967,14 +2967,16 @@ void test_ecmult_multi(secp256k1_scratch *scratch, secp256k1_ecmult_multi_func e
 
 void test_ecmult_multi_batch_single(secp256k1_ecmult_multi_func ecmult_multi) {
     secp256k1_scalar szero;
-    secp256k1_scalar sc[32];
-    secp256k1_ge pt[32];
+    secp256k1_scalar sc;
+    secp256k1_ge pt;
     secp256k1_gej r;
     ecmult_multi_data data;
     secp256k1_scratch *scratch_empty;
 
-    data.sc = sc;
-    data.pt = pt;
+    random_group_element_test(&pt);
+    random_scalar_order(&sc);
+    data.sc = &sc;
+    data.pt = &pt;
     secp256k1_scalar_set_int(&szero, 0);
 
     /* Try to multiply 1 point, but scratch space is empty.*/

